src/test/java/com/wikia/webdriver/testcases/activityfeedstests/WikiActivityTests.java

```diff
@@ -136,6 +134,7 @@ private ArticlePageObject createArticle() {
   private String editArticleWithContentAndGetTitle(ArticlePageObject article, String content) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "editArticleWithContentAndGetTitle" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)